databases/week3/queries3.sql

```diff
@@ -0,0 +1,41 @@
+use meal_sharing;
+
+#Get meals that has a price smaller than a specific price fx 90
+SELECT * FROM meal 
+WHERE price < 90;
+
+#Get meals that still has available reservations
+SELECT meal_id, title, SUM(number_of_guests) AS number_of_guests, max_reservations FROM meal
+LEFT JOIN reservation ON meal.id= reservation.meal_id
+group by meal.id
+HAVING meal.max_reservations > number_of_guests OR number_of_guests IS NULL ;
+
+#Get meals that partially match a title.
+SELECT * FROM meal 
+WHERE title like "%Rød grød med%";
+
+#Get meals that has been created between two dates
+SELECT * FROM meal
+WHERE created_date 
+BETWEEN '2021-03-25' AND '2021-03-27';
+
+#Get only specific number of meals
+SELECT * FROM meal 
+LIMIT 3;
+
+#Get the meals that have good reviews
+SELECT review.meal_id, meal.title, review.stars FROM meal
```

Comment:
Again it's better to select `meal.id` here. It doesn't matter as much when doing an inner join, but it's nice to always follow that rule for consistency :)